sidebars.js

```diff
@@ -179,8 +179,11 @@ module.exports = {
           label: 'Automated Observability',
           collapsed: true,
           items: [
-            'platform-engineers/operations/observability',
-            'platform-engineers/operations/logging',
+            'platform-engineers/observability/quickstart',
```

Comment:
the quickstart doc should have the name `observability`

Reply:
ok